mantidproject/mantid
# Remove message box specificity

**Description of work.**
`message_box.py` is intended to contain a common class: for this reason, the title should not be specific to Frequency Domain Analysis.

**Report to:** nobody

**To test:**
Open Mantid: Interfaces -> Muon -> Frequency Domain Analysis
The message box should have a title of `Error`

Fixes #23066 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
